### PR TITLE
Create maxmind_geo_ip.rst

### PR DESCRIPTION
**Pull request by marjohn56**
Help for new Maxmind GeoIP requirements.

### CODE REVIEW

**Comment by marjohn56** [comment]
Ad - I removed the comment about working without a licence, as I've just been informed it will only work that way until the end of the month,

File: source/manual/how-tos/maxmind_geo_ip.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
+*************************
+**MaxMind GeoIP's Setup**
+*************************
+
+With the changes MaxMind have implemented it's now a requirement that anyone using their lists must have an account and by having that account will have accepted their data protection requirements. It's fairly simple to set-up so let's get started.
```
Comment (fabianfrz) [inline source/manual/how-tos/maxmind_geo_ip.rst]: How about:
```suggestion
With the changes MaxMind have implemented it is now a requirement that anyone using their lists must have an account and complying to their data protection requirements. It's fairly simple to set-up so let's get started.
```

File: source/manual/how-tos/maxmind_geo_ip.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+###################
+Create An Account
+###################
+
+Goto https://www.maxmind.com/en/geolite2/signup and create your account, note the email you provide will be used to send you the link you will need to enter in Opnsense, so make sure its a real account.
```
Comment (fabianfrz) [inline source/manual/how-tos/maxmind_geo_ip.rst]:
```suggestion
Goto https://www.maxmind.com/en/geolite2/signup and create your account. Note that the email address you provide will be used to send you the link you will need to enter in OPNsense, so make sure its a real account.
```

File: source/manual/how-tos/maxmind_geo_ip.rst [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+Now we need to create the link we'll need in Opnsense, all you need to do now is to replace the 'My Licence key' part of the link below with your license key.
+
+https://download.maxmind.com/app/geoip_download?edition_id=GeoLite2-Country-CSV&licence_key=My_Licence_key&suffix=zip
+
+You can check that you have done it correctly by just pasting the link into a browser, it should download the zip file. 
+
+
+##########
+Opnsense
```
Comment (fabianfrz) [inline source/manual/how-tos/maxmind_geo_ip.rst]:
```suggestion
OPNsense
```

File: source/manual/how-tos/maxmind_geo_ip.rst [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+##########
+
+In Opnsense, goto Firewall:Aliases and select the GeoIP settings tab. Enter the URL you have created into the URL box and click Apply, and that's it.
```
Comment (fabianfrz) [inline source/manual/how-tos/maxmind_geo_ip.rst]:
```suggestion
In OPNsense, goto Firewall:Aliases and select the GeoIP settings tab. Enter the URL you have created into the URL box and click Apply, and that's it.
```

Comment (marjohn56) [reply source/manual/how-tos/maxmind_geo_ip.rst]: I'll make the changes you suggested in the morning and squash them  back into one commit.

**Comment by AdSchellevis** [comment]
@marjohn56 I'll merge once I'll have time to take a look at the alias page, where this page should be referred from. Thanks!

**Comment by marjohn56** [comment]
OK, I've also done a couple of minor changes to the xml help on the Geoip page, I was going to put a link there, obviously cannot to that until the wiki page is in place so I'll leave you to do that.

**Comment by AdSchellevis** [comment]
@marjohn56 ok, thanks, I'll take a look. I'll probably replace MaxMind with something like vendor to allow other options as well.

**Pull request merged**